### PR TITLE
pieter_temmerman

### PR DESCRIPTION
**Pull request by ptemmer**

### CODE REVIEW

File: pieter_temmerman/session-01/exercise-02/mariadb-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      containers:
+      - image: bitnami/mariadb:latest
+        name: mariadb
+        envFrom:
+          - configMapRef:
+              name: mariadb-config
+          - secretRef:
+              name: mariadb-secrets
+        livenessProbe:
+          exec:
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-02/mariadb-deployment.yaml]: Nice! Alternative:

```
          exec:
            command:
            - mysqladmin
            - ping
```

File: pieter_temmerman/session-01/exercise-02/wordpress-canary-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  name: wordpress-canary-deployment
+  namespace: exercise-02
+  labels:
+    app: wordpress
+    tier: frontend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-02/wordpress-canary-deployment.yaml]: Very nice!!

File: pieter_temmerman/session-01/exercise-02/wordpress-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  name: wordpress-deployment
+  namespace: exercise-02
+  labels:
+    app: wordpress
+    tier: frontend
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-02/wordpress-deployment.yaml]: I would a 3rd label as you did on Canary so you can also identify them easily on the cluster. For instance:
```
env: stable
```

File: pieter_temmerman/session-01/exercise-02/wordpress-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    spec:
+      containers:
+      - image: bitnami/wordpress:4.9.4
+        name: wordpress
+        envFrom:
+          - configMapRef:
+              name: wordpress-config
+          - secretRef:
+              name: wordpress-secrets
+        livenessProbe:
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-02/wordpress-deployment.yaml]: Nice probes!

File: pieter_temmerman/session-01/exercise-02/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+### How to deploy
+
+Execute 'commands.bash'
+
+Note: It is expected that your kubectl environment is configured correctly.
+That is, it should be pointing to the correct Kubernetes API IP and use the X509 certificates to authenticate against it.
+
+### Curiosities
+
+Strangely enough, you can create an ingress referencing a non-existent tls-secret. Moreover, "kubectl describe ingress NAME" shows it's configured to serve HTTPS requests. This lead to some confusion (and lost time), as I wasn't aware that my tls-secret wasn't properly created.
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-02/README.md]: Totally agree! It should complain when the secret is missing

File: pieter_temmerman/session-01/exercise-03/wordpress-deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: wordpress
+        tier: frontend
+    spec:
+      containers:
+      - image: bitnami/wordpress:4.9.4
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-03/wordpress-deployment.yaml]: Use latest on 3rd exercise

File: pieter_temmerman/session-01/exercise-03/ingress.yaml [hunk 7]
```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: wordpress-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/affinity: cookie
+    nginx.ingress.kubernetes.io/configuration-snippet: |
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-03/ingress.yaml]: Nice!

File: pieter_temmerman/session-01/exercise-03/commands.bash [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+kubectl create -n exercise-03 configmap wordpress-config --from-env-file=wordpress.properties
+
+#Create deployments
+kubectl create -n exercise-03 -f wordpress-deployment.yaml
+
+#Create services
+kubectl create -n exercise-03 -f rds-service.yaml
+kubectl create -n exercise-03 -f wordpress-service.yaml
+
+#Create a and add SSL certificate keypair ()
```
Comment (juan131) [inline pieter_temmerman/session-01/exercise-03/commands.bash]: Perfect!

File: pieter_temmerman/session-01/exercise-01/wordpress.properties [hunk 9]
```diff
@@ -0,0 +1,8 @@
+WORDPRESS_USER=wp_user
```
[the remark this answers is not in the document]
Comment (ptemmer) [reply pieter_temmerman/session-01/exercise-01/wordpress.properties]: Good catch!

File: pieter_temmerman/session-01/exercise-01/ingress.yaml [hunk 10]
```diff
@@ -0,0 +1,14 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: wordpress-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/affinity: cookie
```
[the remark this answers is not in the document]
Comment (juan131) [reply pieter_temmerman/session-01/exercise-01/ingress.yaml]: You can use the default values. It's just a good practise to have control about the name and hash used on our cookies.

**Pull request merged**